how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -99,115 +122,132 @@ $ services=(
 
 $ for s in "${services[@]}"; do 
 
-	service_dir="/etc/systemd/system/${s}.service.d/"
-	sudo mkdir -p ${service_dir}
+    service_dir="/etc/systemd/system/${s}.service.d/"
+    sudo mkdir -p ${service_dir}
 
-	cat << EOT | sudo tee "${service_dir}/proxy.conf"
+    cat << EOT | sudo tee "${service_dir}/proxy.conf"
 [Service]
 Environment="HTTP_PROXY=${http_proxy}"
 Environment="HTTPS_PROXY=${https_proxy}"
 Environment="NO_PROXY=${no_proxy}"
 EOT
 done
+
 $ sudo systemctl daemon-reload
 ```
 
-###  Start Kubernetes with kubeadm 
+## Start Kubernetes
 
-```bash
-# Mark sure containerd is up and running
-$ sudo systemctl restart containerd
-$ sudo systemctl status containerd
+- Make sure `containerd` is up and running
 
-# Prevent docker iptables rules conflict with k8s pod communication
-$ sudo iptables -P FORWARD ACCEPT
+  ```bash
+  $ sudo systemctl restart containerd
+  $ sudo systemctl status containerd
+  ```
 
-# Start cluster using kubeadm
-$ sudo kubeadm init --skip-preflight-checks \
---cri-socket /run/containerd/containerd.sock --pod-network-cidr=10.244.0.0/16
+- Prevent conflicts between `docker` iptables (packet filtering) rules and k8s pod communication
 
-$ export KUBECONFIG=/etc/kubernetes/admin.conf
+  ```bash
+  $ sudo iptables -P FORWARD ACCEPT
```

Comment:
Ideally I'd like a touch more info around what this is fixing, how, and why it's not a gaping security hole (or, a caveat note if it is ;-) ).

Reply:
Well, this PR is for reworking the formatting but I'm happy to take input from @jcvenegas with more details... :)

Reply:
@jodh-intel  this is used for the case that docker is installed in the node, docker set iptables rules that not allow containers communication, the issue is  https://github.com/kubernetes/kubernetes/issues/40182

Reply:
I did not test if this is not needed anymore, but keep as legacy from our initial k8s installation script.

Reply:
Thanks @jcvenegas - branch updated.

Reply:
@grahamwhaley, @jcvenegas  - can you provide some suitable "security hole or not" words I can add?

/cc @mcastelino.

Reply:
We didn't resolve this. I'm not happy with it as is, as it applies a broad spectrum port forwared/allow, which is then not sticky. We need a decision. /cc @mcastelino @egernst .